src/loaders/js.ts

```diff
@@ -48,7 +48,7 @@ export const jsLoader: Loader = async (input, { options }) => {
   // esm => cjs
   const isCjs = options.format === "cjs";
   if (isCjs) {
-    contents = jiti("")
+    contents = createJiti("")
       .transform({ source: contents, retainLines: false })
```

Comment:
CJS Transform has breaking changes which makes it incompatible, we can't upgrade.

Reply:
Oh, sorry for not being thorough. I'll set the PR to “draft” for now and try to look at it again soon.

Reply:
Thanks. FWIW i think we can eventually migrate to newer obuild stack. there is an ongoing PR to add similar mkdist functionaliy to it (https://github.com/unjs/obuild/pull/34)